gcloud/dns/zone.py

```diff
@@ -220,9 +228,11 @@ def _build_resource(self):
         """Generate a resource for ``create`` or ``update``."""
         resource = {
             'name': self.name,
-            'dnsName': self.dns_name,
         }
 
+        if self.dns_name is not None:
+            resource['dnsName'] = self.dns_name
```

Comment:
Maybe it should've been in #1728, which allowed `dns_name` to be None;  the coverage gap didn't surface until here (because `dns_name` is now (as of this PR) the fallback for `description`, and because it is now allowed to be `None`).